src/helpers.ts

```diff
@@ -244,20 +244,22 @@ export async function PlaySpecificTracks(
 	instance: SpotifyInstanceBase,
 	deviceId: string,
 	uris: string[],
-	attempt = 0
+	attempt = 0,
+	positionMs = 0
```

Comment:
This parameter should be before attempt, probably without a default value.

Reply:
@Julusian updated